### DISCUSSION TITLE
Could you add Finnish Easylist Addition

### DISCUSSION THREAD

**Discussion by Raitsa** [AdguardTeam/AdGuardHome#5576] [General]
Could you add Finnish Easylist Addition to the available blocklists, please :)
 
https://github.com/finnish-easylist-addition/finnish-easylist-addition

**Comment by ainar-g** [comment]
Hello.  Please open an issue about that in the <https://github.com/AdguardTeam/HostlistsRegistry> repository, if there isn't one already.

**Comment by Raitsa** [comment]
Thanks, done. https://github.com/AdguardTeam/HostlistsRegistry/issues/184
